src/components/ContentWrapper.jsx

```diff
@@ -62,10 +61,44 @@ const ContentWrapper = (props) => {
   const user = useSelector((state) => state.user.current);
 
   const samples = useSelector((state) => state.samples);
-
   // selectedExperimentID holds the value in redux of the selected experiment
```

Comment:
can you move this comment to where it is relevant? (after the new function you wrote)